BasicTests/genesishashestest.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
I have no idea how many teams are still using this files. perhaps it is better to leave it here as it is small